README.md

```diff
@@ -36,3 +37,16 @@ constant factors.
 Describe your reasoning and the conclusion you've come to. Your reasoning is
 most important -- you can easily find the answer, but you need to demonstrate
 that you've understood the concept. Add your answer to this markdown file.
+
+## Average-Case Time Complexity Answer
+
+### Looking deeper at best and worst case
+
+Looking deeper at the best and worst case I can get a good idea of what is between, or our average case here. </br>
+
+The best possible case would be if the list is already sorted in ascending order. The function would run through only the first `for` block and would therfore have a complextity of $\theta(n)$. 
+In the worse case, where it's sorted in descending order, we have to run through both blocks the entire way through the array, doing the same amount of work each time. So the complexity would be $\theta(n^2)$. 
+
+### The Average case
+
+So now, our best and worst case can be our bounds for the average case. Critically thinking, if there is ever a number in the wrong place, we have to run both `for` blocks and immediatly get out of the $\theta(n)$ complexity. Likewise, the only time we have a complexity of $\theta(n^2)$ is in the worst possible case, if we have any number that is not in descending order, the nested `for` loop would not work as hard in that case as it would with the other numbers and not be truly be $n * n$. So the average case time complexity would have to be $\theta(n\log n)$.
```

Comment:
Wait, so the average case is better than for the non-reverse insertion sort?

Reply:
Right, we have no idea how much work the nested `for` loop would do with each iteration, it's not that simple.